src/main_dpdk.cpp

```diff
@@ -5951,6 +5978,24 @@ COLD_FUNC int update_global_info_from_platform_file(){
         }
     }
 
+    if (cg->m_latency_measurement.length()) {
```

Comment:
I think [empty()](http://www.cplusplus.com/reference/string/string/empty/) is more suitable.

Reply:
Again, I was trying to keep the file more coherent (see L992, L6271, L6277, L6281)

Reply:
Understood, but IMHO, we should express our intention in most explicite way. If we want to check if string is not empty, we should check if `! .empty()`.
But it is just me.